source/hardware/controllers/native_instruments_traktor_kontrol_s2_mk3.rst

```diff
@@ -32,108 +32,110 @@ Mixxx mapping
+| **14**. Shift      | Activates secondary functions when pressed                                                                                  |                                                     |
+| button             |                                                                                                                             |                                                     |
++--------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
+| **15**. CUE button | CUE default                                                                                                                 | If the CUE point is set, jump to it and stops       |
++--------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
+| **16**. Play button| Toggles playing                                                                                                             | Seeks a player to the start and then stops it       |
++--------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
+| **17**. Number     | Function depends on current mode                                                                                            |                                                     |
+| buttons            |                                                                                                                             |                                                     |
++--------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
+| Hotcue mode        | If hotcue is set, seeks the player to hotcue position. Otherwise set hotcue at current position                             | Clear the hotcue                                    |
++--------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
+| Samples mode       | If track is loaded into corresponding slot, go to CUE point and play                                                        | If track is playing, CUE default behaviour.         |
+|                    |                                                                                                                             | Otherwise eject track                               |
```

Comment:
Does pressing the button load the selected track?
Just wondering, because Shift+press can eject, and if that can be done why not also load.